lnbits/extensions/boltcards/README.md

```diff
@@ -42,18 +42,23 @@ Updated for v0.1.3
     - Or you can Click the "KEYS / AUTH LINK" button to copy the auth URL to the clipboard. Then paste it into the Android app (Create Bolt Card -> PASTE AUTH URL).
 - Click WRITE CARD NOW and approach the NFC card to set it up. DO NOT REMOVE THE CARD PREMATURELY! 
 
-## Erasing the card - Boltcard NFC Card Creator
-Updated for v0.1.3
+## Rewriting / Erasing the card - Boltcard NFC Card Creator
+
+It is possible not only to reset the keys but also to disable the SUN function and completely erasing the card so it can be used again as a static tag or set up as a new Bolt Card. 
```

Comment:
```suggestion
It is possible not only to reset the keys but also to disable the SUN function and completely erase the card so it can be used again as a static tag or set up as a new Bolt Card. 
```